include/ginkgo/core/stop/criterion.hpp

```diff
@@ -269,6 +277,8 @@ using EnableDefaultCriterionFactory =
  *                          and the public getter's name is
  *                          `get_<_parameters_name>()`)
  * @param _factory_name  name of the generated factory type
+ *
+ * @ingroup stop
```

Comment:
Same here, maybe `@ingroup` the namespace if it propagates the information to internal objects. Maybe the macro does need the `@ingroup` though.